README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
```

Comment:
What about enums? Should enums be in the same file as the one they're in (which is common now in both objc and Swift) or should they get their own separate file too? What about protocols?

Also, this rule can't be adhered to if you wish to declare a private class. I think this needs more expansion.